app/models/ui/RowLists.scala

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
+// $COVERAGE-OFF$
+
 package models.ui
 
-case class SummaryCell(txt: Text)
-case class SummaryRow(cells: Seq[SummaryCell])
-case class SummaryList(rows: Seq[SummaryRow]) extends UIComponent {
+case class SummaryList(rows: Seq[Seq[Text]]) extends UIComponent {
   val text: Text = Text()
```

Comment:
It would be better if we didn't have to include the probably unused attribute text. Perhaps we can investigate how much effort this would require sometime.

Reply:
yes. If the heading is included in the SumaryList component (which possibily it should be), text could reference this, which is why text was originally included in UIComponent (see heading on the ui page classes). will look into it

Reply:
Having looked into this, the SummaryList pattern in GDS doesnt need a heading